miio/fan_miot.py

```diff
@@ -525,3 +548,260 @@ def delay_off(self, minutes: int):
             raise FanException("Invalid value for a delayed turn off: %s" % minutes)
 
         return self.set_property("power_off_time", minutes)
+
+
+class OperationModeFanZA5(enum.Enum):
+    Nature = 0
+    Normal = 1
+
+
+class FanStatusZA5(DeviceStatus):
+    """Container for status reports for FanZA5."""
+
+    def __init__(self, data: Dict[str, Any]) -> None:
+        self.data = data
+
+    # TODO: Docstrings.
+    @property
+    def anion(self) -> bool:
```

Comment:
What does this mean?

Reply:
It is the state of a built-in [air ionizer](https://en.wikipedia.org/wiki/Air_ioniser):
> Some commercial air purifiers are designed to generate negative ions.

Reply:
Maybe `is_ionizer_enabled` then?

Reply:
Renamed as suggested.